### PR TITLE
Directory structure changes for pypi. Docs Added.

### PR DESCRIPTION
**Pull request by Govind9**

### CODE REVIEW

File: waterwheel/hydromatch.py [hunk 1]
```diff
@@ -85,37 +87,35 @@ def load_vocab(self, water_bodies: Dict, append: bool = True):
...
             self.matcher.add(key.upper(), None, *phrases)
             if key not in self.wikidata:
                 self.wikidata[key] = {}
             for name, id in water_bodies[key]:
                 self.wikidata[key][name.lower()] = id
-            print (f"{len(phrases)} {key}(s) added.")
-    
-    def save_vocab(self, filename = "hydro_matcher.pkl"):
+
+    def save_vocab(self, filename = Path('resources/hydromatch.pkl')):
```
Comment (lintool) [inline waterwheel/hydromatch.py]: Method of saving data should probably go in a separate file...

File: waterwheel/hydromatch.py [hunk 2]
```diff
@@ -1,73 +1,75 @@
...
             Path to the file with lake data
         append : bool, optional
             If True then the new data is appended on top of the previous data.
             Otherwise old data is done away with.
         """
-        water_bodies = {"LAKE": [], "RIVER": []}
+
+        water_bodies = {'LAKE': [], 'RIVER': []}
         try:
             dfr = pd.read_csv(river_csv)
```
Comment (lintool) [inline waterwheel/hydromatch.py]: this should also be moved to a util script somewhere?

**Comment by lintool** [comment]
I think the API we want is something like:

```python
from waterwheel import WaterWheel
ner = WaterWheel(...)
doc = nlp(...)
```

So our code behaves just like any other spaCy module?

https://spacy.io/api/entityrecognizer

**Comment by Govind9** [comment]
Going the spacy example shouldn't we go with
```python
from waterwheel.matcher import WaterWheel
```

> I think the API we want is something like:
> 
> ```python
> from waterwheel import WaterWheel
> ner = WaterWheel(...)
> doc = nlp(...)
> ```
> 
> So our code behaves just like any other spaCy module?
> 
> https://spacy.io/api/entityrecognizer

**Comment by lintool** [comment]
I don't think our package is complex enough to warrant subpackages yet.. so let's stay with

```python
from waterwheel import WaterWheel
```

**Pull request closed without merging**